src/transaction/combine.ts

```diff
@@ -1,42 +1,80 @@
+  const tx = transactions[0];
+  const _txSignersTemp = tx.Signers;
+  delete tx.Signers
+  
+  transactions.forEach( _tx => {
+      const _txSignersTemp = _tx.Signers
+      delete _tx.Signers;
+      
+      if ( JSON.stringify(tx) !== JSON.stringify(_tx)) {
+          throw new ValidationError('txJSON is not the same for all signedTransactions')
+      }
+
+      _tx.Signers = _txSignersTemp
+  })
+
+  tx.Signers = _txSignersTemp
+  return transactions
+}
 
 function addressToBigNumber(address) {
   const hex = Buffer.from(decodeAccountID(address)).toString('hex')
   return new BigNumber(hex, 16)
 }
 
+/**
+ * According to the documentation on multi-signatures:
+ * If presented in binary form, the Signers array must be sorted based on 
+ * the numeric value of the signer addresses, with the lowest value first. 
+ * (If submitted as JSON, the submit_multisigned method handles this automatically.)
+ */
 function compareSigners(a, b) {
   return addressToBigNumber(a.Signer.Account).comparedTo(
     addressToBigNumber(b.Signer.Account)
   )
 }
 
-function combine(signedTransactions: Array<string>): object {
-  validate.combine({signedTransactions})
-
-  // TODO: signedTransactions is an array of strings in the documentation, but
-  // tests and this code handle it as an array of objects. Fix!
-  const txs: any[] = _.map(signedTransactions, binary.decode)
-  const tx = _.omit(txs[0], 'Signers')
-  if (!_.every(txs, (_tx) => _.isEqual(tx, _.omit(_tx, 'Signers')))) {
-    throw new utils.common.errors.ValidationError(
-      'txJSON is not the same for all signedTransactions'
-    )
-  }
+function getATransactionWithAllSigners(transactions : Array<JsonObject>) : JsonObject {
   const unsortedSigners = _.reduce(
-    txs,
+    transactions,
     (accumulator, _tx) => accumulator.concat(_tx.Signers || []),
     []
   )
-  const signers = unsortedSigners.sort(compareSigners)
-  const signedTx = _.assign({}, tx, {Signers: signers})
-  const signedTransaction = binary.encode(signedTx)
+  //The documentation requires signers to be sorted
+  const sortedSigners = unsortedSigners.sort(compareSigners)
+
+  //This is used to make a new copy of the transaction with ALL signers
+  const signedTx = _.assign({}, transactions[0], {Signers: sortedSigners})
+
+  return signedTx
+}
+
+/**
+ * 
+ * @param signedTransactions A collection of the same transaction signed by different signers. The only difference
+ * between the elements of signedTransactions should be the Signers field.
+ * @returns An object with the combined transaction (now having a sorted list of all signers) which is encoded, along
+ * with a transaction id based on the combined transaction.
+ */
+function combine(signedTransactions: Array<string>): object {
+  validate.combine({signedTransactions})
+
+  const transactions: JsonObject[] = signedTransactions.map(binary.decode);
+  validateAllSignedTransactionsAreEqual(transactions)
```

Comment:
if you're not using the return value of `transactions` then don't return anything from this. updated proposed function above.